README.md

```diff
@@ -432,6 +432,47 @@ trait_must_use_added = "allow"
 union_must_use_added = "allow"
 ```
 
+#### Configuration Priority
+
+When there are multiple conflicting entries for a given lint (for example, configuring the
+`must_use_added` group containing the `function_must_use_added` lint and `function_must_use_added` itself),
+`cargo-semver-checks` needs more information to determine which configuration entry takes
+precedence. Like the Cargo `[lints]` table, each configuration entry has an optional
+`priority` field that defines the order of configuration entries. If an entry has no
+`priority` key, its priority is 0 by default. Entries with a lower (more negative)
+`priority` override those with a higher `priority` when there is a conflict.
```

Comment:
Consider explaining first, then providing an example, rather that doing both at once. Weaving lint groups and a specific lint into the middle of the explanation leaves readers with a lot of information to track.

Consider adapting the explanation I used below if you decide you like it. Something along the lines of "for each setting in each lint, we inspect config items in priority order and pick the first one that sets a value for that setting."